src/Components/Locations.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+import './Locations.css'
+import { Component } from 'react';
+
+class Locations extends Component {
+    state = {
+        locations: [],
+        showLocations: false
+    }
+
+    getLocations = async () => {
+        try {
+            const { data } = await axios.get(
+                'https://ghibliapi.herokuapp.com/locations'
+            )
+            console.log(data)
+            this.setState({
+                locations: data,
+                showLocations: !this.state.showLocations
```

Comment:
Is your idea here to show the locations automatically once they've been fetched? If so, consider this instead:

`this.setState({
  ...
  showLocations: true
})`